build/tasks/aot.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+var gulp = require("gulp");
+var inlineNg2Template = require("gulp-inline-ng2-template");
+var runSequence = require('run-sequence');
+var del = require("del");
+var os = require('os');
+
+gulp.task("aot:copy", function() {
+    var claritySources = [
+        'src/clarity-angular/**/*.ts',
+        'src/clarity-angular/**/*.html',
+        '!src/clarity-angular/**/*.spec.ts',
+        '!src/clarity-angular/**/*.mock.ts'
+    ];
+
+    gulp.src(claritySources)
+        .pipe(inlineNg2Template({
```

Comment:
If we opt to not inline the templates, we need to include our template files (which we haven't been) in our npm packages. Do we want to do that?